packages/site-components/src/VerticalNavigation.tsx

```diff
@@ -54,7 +54,8 @@ const createMenuItemStyles = colorMode => ({
 });
 
 const rootStyles = {
-  borderRightWidth: '0px'
+  borderRightWidth: '1px',
```

Comment:
the problem with the border is that it exposes a new issue, that our footer is not full width.. we need to refactor layouts, if design folks want a full width footer?

That might not be something we want to do right now.

You do not need to set this as it is the default, the previous edit, turned it off, remove line 57 and it will be back.

Another new issue of adding the border is that it's the pro-sidebar default grey, so we would need to add the variables like we have done for the highlight color. The border changes color between light and dark mode.

Our theme has variables for border colors, so we could use those.